src/assets/i18n/en/common.json

```diff
@@ -0,0 +1,20 @@
+{
+  "QUEUE": {
+    "QUEUE_TITLE": "Machine Actions in Queue",

```

Comment:
It's better to change it to TITLE. In this case translation use will look like `QUEUE.TITLE`